app/src/main/java/ru/practicum/android/diploma/details/presentation/ui/DetailFragment.kt

```diff
@@ -135,7 +135,7 @@ class DetailFragment : Fragment() {
 
         binding.progress.isVisible = false
         binding.scroll.isVisible = true
-        binding.phDetailsError.isVisible = false
+        binding.include.root.isVisible =  false
```

Comment:
🍏 Рекомендую давать вьюхам более развернутые понятные названия. Например,
```suggestion
        binding.errorPlaceholder.root.isVisible =  false
```